Add a 'flat' profile mode (#228)
vmprof/vmprof-python: README.md

Setting up development can be done using the following commands:

>         $ virtualenv -p /usr/bin/python3 vmprof3
>         $ source vmprof3/bin/activate
>         $ python setup.py develop

Reviewer: Nit: 4 spaces not 8?
```suggestion
    $ virtualenv -p /usr/bin/python3 vmprof3
    $ source vmprof3/bin/activate
    $ python setup.py develop
```

```

### Line-based output

vmprof supports line profiling mode, which enables collecting and showing the statistics for separate lines
inside functions.

To enable collection of lines statistics add `--lines` argument to vmprof:

```console
$ python -m vmprof --lines -o <output-file> <your program> <your program args>
```

Or pass `lines=True` argument to `vmprof.enable` function, when calling vmprof from code.

> To see line statistics for all functions add the `--lines` argument to `vmprofshow`:

Reviewer: ```suggestion
To see line statistics for all functions use the  `lines` mode of `vmprofshow`:
```

    17      297     25.4          return self._rand
```

### "Flattened" output

> 

Reviewer: ```suggestion
`vmprofshow` also has a `flat` mode.

```
